components/FAQ.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div>
+    <div v-for="faq in listOfFaq" :key="faq.id">
+      <br />
+      <input type="checkbox" id="checkbox" v-model="faq.data.selected" />
+      Question:
+      {{ faq.data.question }}
+      <br />
+      Category:
+      {{ faq.data.category }}
+      <br />
+      Answer:
+      {{ faq.data.answer }}
+      <br />
+      Timestamp:
+      {{ faq.data.timestamp }}
```

Comment:
Just curious what will be displayed here when timestamp is null?
I can't see it in data object, and it seems like "firebaseTimestamp" will be the key that assigned to the data (if i read ...firebaseTimestamp properly).